asApp/src/os/Linux/osdNfs.c

```diff
@@ -22,20 +22,27 @@ extern volatile int save_restoreDebug;
 int nfsMount(char *uidhost, char *path, char *mntpoint)
 {
     struct stat st;
-    int         devl;
-    char       *host;          /* host string */
-    int         rval = -1;     /* return value */   
-    char       *dev  =  0;     /* buffer for the full source path */
+    int devl;
+    char *host;    /* host string */
+    int rval = -1; /* return value */
+    char *dev = 0; /* buffer for the full source path */
 
     /* check inputs */
     if (!uidhost || !path || !mntpoint) {
-        fprintf(stderr,"usage: nfsMount(""[uid.gid@]host"",""path"",""mountpoint"")\n");        
+        fprintf(stderr,
+                "usage: nfsMount("
+                "[uid.gid@]host"
+                ","
+                "path"
+                ","
+                "mountpoint"
+                ")\n");
```

Comment:
Instead of inserting a newline between each `""` in the original string here, I would have replaced every `""` with `\"` since I think that's what the original author probably meant:
`"usage: nfsMount(\"[uid.gid@]host\",\"path\",\"mountpoint\")\n"`

Reply:
True, this seems to be the intention. Done.